src_assets/linux/misc/85-sunshine.rules

```diff
@@ -1 +1 @@
-KERNEL=="uinput", GROUP="input", MODE="0660", OPTIONS+="static_node=uinput"
\ No newline at end of file
+KERNEL=="uinput", SUBSYSTEM=="misc", GROUP="input", MODE="0660", OPTIONS+="static_node=uinput", TAG+="uaccess"
```

Comment:
Isn't the input devices group and mode redundant then and can be left unchanged?
```suggestion
KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess"
```
Or do you want to keep it for mentioned cases with users created in containers where it may still be needed?

Reply:
Yes, the original intention was to keep group and mode for special cases, such as users created in Docker containers. Regular Linux users don't need group/mode configuration. `TAG+="uaccess"` already handles that.

But as discussed above, Sunshine container also doesn't seem to need additional group/mode configuration because it passes UID of host user to container. So `GROUP="input", MODE="0660"` seem redundant in this particular case as well.

Should I remove group/mode from udev rule and fully rely on `TAG+="uaccess"`? This is basically exactly what Steam did:

https://github.com/ValveSoftware/steam-devices/blob/master/60-steam-input.rules#L5

Reply:
> Should I remove group/mode from udev rule and fully rely on `TAG+="uaccess"`? This is basically exactly what Steam did:

Seems reasonable to not override distro and possibly admin decisions about these UNIX permissions if Sunshine is not commonly make us of them anymore. However, let Sunshine maintainers comment on this first, as I would understand the argument to leave it in place for another testing/feedback iteration.

Reply:
Are there any negative effects leaving `GROUP="input", MODE="0660"` there?

Reply:
Not necessarily. But distributors and admins will be thankful when the changes to the system are limited to what is really needed.

Reply:
I guess let's remove it then and we can put it back if we find issues.

Reply:
Done. I can think of only one negative effect. If a distro doesn't provide `input` group, udev might complain about it in system logs. Popular distros seem to have `input` group but it's better not to rely on it.